src/lib/components/inputs/FileInput.svelte

```diff
@@ -38,19 +43,39 @@
   ) => {
     onChange(event, maxFileSize, supportedFileTypes);
   };
+
+  const formatSizeUnits = (bytes: number) => {
+    let response: string;
+    if (bytes >= 1073741824) {
```

Comment:
Can we take this from constant which you may define at the top of the file

Reply:
resolved